packages/runtime-core/src/components/Teleport.ts

```diff
@@ -406,6 +406,21 @@ function hydrateTeleport(
     }
     updateCssVars(vnode)
   }
+  if (!target && isTeleportDisabled(vnode.props)) {
```

Comment:
```suggestion
  else if (isTeleportDisabled(vnode.props)) {
    if (vnode.shapeFlag & ShapeFlags.ARRAY_CHILDREN) {
      vnode.anchor = hydrateChildren(
        nextSibling(node),
        vnode,
        parentNode(node)!,
        parentComponent,
        parentSuspense,
        slotScopeIds,
        optimized,
      )
      vnode.targetStart = node
      vnode.targetAnchor = nextSibling(node)
    }
  }
```

Reply:
Could you please add a test case?

Reply:
Thank you for your review, I will improve it